docs/user/managing-devices.md

```diff
@@ -88,33 +105,33 @@ apiVersion: flightctl.io/v1alpha1
 kind: Device
 metadata:
   name: 54shovu028bvj6stkovjcvovjgo0r48618khdd5huhdjfn6raskg
-  labels:                                    # <-- user-defined labels assigned to this device
+  labels: # <-- user-defined labels assigned to this device
     region: eu-west-1
     site: factory-berlin
 spec:
   os:
     image: quay.io/flightctl/rhel:9.5        # <-- the device's target OS image version
   config:
-  - name: my-os-configuration                # <-- the device's target OS configuration (here: read from a git repo)
-    configType: GitConfigProviderSpec
-    gitRef:
-      path: /configuration
-      repository: my-configuration-repo
-      targetRevision: production
+    - name: my-os-configuration                # <-- the device's target OS configuration (here: read from a git repo)
+      configType: GitConfigProviderSpec
+      gitRef:
+        path: /configuration
+        repository: my-configuration-repo
+        targetRevision: production
 status:
   os:
     image: quay.io/flightctl/rhel:9.5        # <-- the device's current OS image version
   config:
     renderedVersion: "1"                     # <-- the device's current OS configuration version
   applications:
-    data: {}                                 # <-- the device's current list of deployed applications
+    data: { }                                 # <-- the device's current list of deployed applications
     summary:
       status: Unknown                        # <-- health status of applications on the device
-  resources:                                 # <-- whether sufficient CPU/disk/memory resources are available
+  resources: # <-- whether sufficient CPU/disk/memory resources are available
```

Comment:
it seems a lot of the changes to this file are cosmetic. could we consider splitting this into a separate cleanup pr? i think a lot of the changes make sense, but some don’t seem needed.

Reply:
as long as the document is formatted the same throughout lgtm